config/jobs/cert-manager/cert-manager/cert-manager-presubmits.yaml

```
@@ -36,6 +36,38 @@ presubmits:
           - name: ndots
             value: "1"
 
+  - name: pull-cert-manager-make-test
+    always_run: false
+    optional: true
+    context: pull-cert-manager-make-test
+    max_concurrency: 8
+    agent: kubernetes
+    decorate: true
+    branches:
+    - master
+    - release-1.8
+    annotations:
+      testgrid-create-test-group: 'true'
+      testgrid-dashboards: jetstack-cert-manager-presubmits-blocking
+      description: Runs 'bazel test --jobs=1 //...'
+    labels:
+      preset-service-account: "true"
+    spec:
+      containers:
+      - image: 1.17.8-alpine@sha256:e2e68a9cdd5da82458652fdac3908a3a270686b38039f2829855398e2e06019d
+        args:
+        - runner
+        - make
+        - test
```

suggestion: I suggest the `test-ci` target since that'll generate JUnit XML output which we can wire up to prow for test output (e.g. in [pages like this](https://prow.build-infra.jetstack.net/view/gs/jetstack-logs/pr-logs/pull/cert-manager_cert-manager/4925/pull-cert-manager-bazel/1500889334834794496)). That might take some plumbing to get it working but I think this is the right target in any case.

```suggestion
        - runner
        - make
        - -f
        - make/Makefile
        - test-ci
```

---

Thanks. This is done in a9e6708